package-lock.json

```diff
@@ -1576,7 +1576,6 @@
       "requires": {
         "anymatch": "1.3.2",
         "async-each": "1.0.1",
-        "fsevents": "1.1.3",
```

Comment:
I'm confused why all these lines got deleted. What version of node are you using?